source/docs/romi-robot/index.rst

```diff
@@ -0,0 +1,15 @@
+Getting Started with Romi
+=========================
+
+.. warning:: We are working hard on getting the Romi documentation complete. Please check back later.
+
+.. image:: images/romi.png
+   :alt: Romi Robot
+
+The Romi is a small and inexpensive robot designed for learning about programming FRC robots. All the same tools used for programming full-sized FRC robots can be used to program the Romi. The Romi comes with two drive motors both with wheel encoders and an IMU sensor that can be used for measuring headings and accelerations. Using it is as simple as writing a robot program, and running it on your computer. It will command the Romi to follow the steps in the program.
```

Comment:
```suggestion
The Romi is a small and inexpensive robot designed for learning about programming FRC robots. All the same tools used for programming full-sized FRC robots can be used to program the Romi. The Romi comes with two drive motors. Included is both wheel encoders and an IMU sensor that can be used for measuring headings and accelerations. Using it is as simple as writing a robot program, and running it on your computer. It will command the Romi to follow the steps in the program.
```

Reply:
English is a bit rough here